src/components/Sticky/index.vue

```diff
@@ -1,8 +1,10 @@
 <template>
```

Comment:
Try to revert changes for this file, and run `yarn serve` to see the result, it should work already, you don't need to change this file.